articles/other/other-ref-idam.md

```diff
@@ -20,6 +20,15 @@ UKCloud's Identity & Access Management API (IDAM) enables you to manage user acc
 
 Full documentation for the IDAM API is available on [SwaggerHub](https://app.swaggerhub.com/apis/ukcloud/idam-service).
 
+## Platforms using IDAM
```

Comment:
We tend to reserve the word platform to refer to the UKCloud platform as a whole. Could we change this to:

UKCloud services using IDAM